src/components/ConnectWalletPaper.tsx

```diff
@@ -82,25 +82,25 @@ export const Disclaimers = () => {
       (
       <Trans>
         By using this Site, I have read and agree to the{' '}
-        <a href="https://www.sparkprotocol.io/terms-of-use.html" target="blank">
+        <a href="https://www.spark.fi/terms-of-use.html" target="blank">
```

Comment:
Are those proper links? They're not working for me.

Reply:
<img width="1063" alt="image" src="https://github.com/marsfoundation/spark-interface/assets/94108039/4dbd44b0-ba63-44b4-9313-7dd88679fcd0">

weird, it has to do with "www"

Reply:
The urls should be updated to use the apex domain.